cfpb/consumerfinance.gov
# TCCP: Add sharing details

Add a description and social sharing image to all the TCCP pages for nicer looking link cards on Facebook, Twitter/X, LinkedIn, etc.

I followed conventions for setting and overriding descriptions and OG/Twitter image in [v1/layouts/base.html](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/jinja2/v1/layouts/base.html#L80-L94), [wellbeing/home.html](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/wellbeing/jinja2/wellbeing/home.html#L15-L19), and [regulations3k/browse-regulation.html](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/regulations3k/jinja2/regulations3k/browse-regulation.html#L58-L60). Happy to change to another method if we have a better way to set any of the sharing details.

---

## Additions

- meta description
- og description
- og/twitter image

## How to test this PR

1. Check all the pages in the TCCP app to make sure their `head`s contain the correct meta/og description and image
    - You might want to check a couple of card detail pages for a card that has "Card" in the product name and for a card that doesn't to make sure the description reads correctly

## Notes and todos

- Once this is on Beta, we'll preview the sharing card on [Facebook](https://developers.facebook.com/tools/debug/) and [Twitter/X](https://cards-dev.twitter.com/validator) to make sure everything looks good.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
